packages/admin-ui/src/pages/packages/components/Network/PortsByService.tsx

```diff
@@ -120,59 +120,101 @@ export function PortsByService({
     return conflictingPorts;
   }
 
-  function getPortsOverTheMax(): PortMapping[] {
+  // Function to get any container ports in the ephemeral range, excluding port 51820
+  function getContainerPortsInEphemeralRange(): PortMapping[] {
     return ports.filter(
-      ({ host, container, deletable }) =>
-        (deletable &&
-          (container > maxPortNumber || (host && host > maxPortNumber))) ||
-        (host && host > maxEphemeralPortNumber)
+      ({ container, deletable }) =>
+        deletable && container > maxRegisteredPortNumber && container <= maxPortNumber && container !== 51820
     );
   }
 
+  // Function to get any host ports in the ephemeral range, excluding port 51820
+  function getHostPortsInEphemeralRange(): PortMapping[] {
+    return ports.filter(
+      ({ host, deletable }) =>
+        deletable && host !== undefined && host > maxRegisteredPortNumber && host <= maxPortNumber && host !== 51820
```

Comment:
I would also create a function called `isInEphemeralRange` that would be called from `getContainerPortsInEphemeralRange` and `getHostPortsInEphemeralRange`